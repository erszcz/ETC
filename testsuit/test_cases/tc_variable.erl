-module(tc_variable).
-compile(export_all).

-spec f1(boolean) -> integer().
f1(X) -> 42.
