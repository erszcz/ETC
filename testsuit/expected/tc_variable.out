Module tc_variable: 
Same as type spec defined in lines ["4"] 
  {f1,1} :: (atom)->integer

test_cases/tc_variable.erl:2: Warning: export_all flag enabled - all functions will be exported
test_cases/tc_variable.erl:5: Warning: variable 'X' is unused
